.github/workflows/maths-jit-diff.yml

```diff
@@ -0,0 +1,112 @@
+name: JIT Disassembly
+
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+env:
+  DOTNET_NOLOGO: 1
+
+jobs:
+  check-comment:
+    outputs:
+      triggered: ${{ steps.check.outputs.triggered }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: khan/pull-request-comment-trigger@master
+        id: check
+        with:
+          trigger: '@jit-asm'
+          reaction: rocket
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+  build-sample-project:
+    runs-on: ubuntu-latest
+    
+    needs: check-comment
+    if: needs.check-comment.outputs.triggered == 'true'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with: 
+          dotnet-version: 5.0.100-preview.7.20366.6
+      - name: Build Project
+        run: dotnet build ./src/Maths/JITTest/JITTest.csproj -f net5.0 -c Release -o ./sample/
+      - uses: actions/upload-artifact@v2
+        with:
+          name: sample
+          path: sample/
+            
+  windows-setup-clr:
+    strategy:
+      matrix:
+        arch: [x64, x86]
+        
+    runs-on: windows-latest
+    
+    needs: check-comment
+    if: needs.check-comment.outputs.triggered == 'true'
+    
+    steps:
+      - name: Cache Artifacts
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./artifacts/
+          key: windows-${{ matrix.arch }}-clr-cache-v5.0.0-preview.7.20364.11 
```

Comment:
Is it possible to move the version into some sort of variable so that we don't need to change it in 2 places.